avail-intellij-plugin/src/main/kotlin/org/availlang/intellij/plugin/module/CreateAvailProjectPanel.kt

```diff
@@ -0,0 +1,185 @@
+/*
+ * CreateProjectPanel.kt
+ * Copyright © 1993-2023, The Avail Foundation, LLC.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ *  * Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ *
+ *  * Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ *  * Neither the name of the copyright holder nor the names of the contributors
+ *    may be used to endorse or promote products derived from this software
+ *    without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.availlang.intellij.plugin.module
+
+import avail.anvil.components.ComboBoxWithLabel
+import avail.anvil.components.TextFieldWithLabel
+import avail.anvil.environment.GlobalEnvironmentSettings
+import avail.anvil.environment.availStandardLibraries
+import com.intellij.util.ui.JBUI
+import org.availlang.artifact.environment.project.*
+import java.awt.GridBagConstraints
+import java.awt.GridBagLayout
+import java.io.File
+import javax.swing.JCheckBox
+import javax.swing.JComboBox
+import javax.swing.JPanel
+
+/**
+ * A [JPanel] used to create a new [AvailProject].
+ *
+ * Adapted to IDEA from [avail.anvil.manager.CreateProjectPanel]
+ *
+ * @author Raul Raja
+ *
+ * @property config
+ *   The [GlobalEnvironmentSettings] for this machine.
+ * @property onCreate
+ *   The function that accepts the newly created [AvailProject].
+ * @property onCancel
+ *   The function to call if creating a new project is canceled.
```

Comment:
None of these fields appear to exist in `CreateAvailProjectPanel`; looks like a cloning error from `avail.anvil.manager.CreateProjectPanel`. Please remove these comments.